egs/vivos/asr1/RESULT

```diff
@@ -22,4 +22,39 @@ exp/train_nodev_pytorch_train_mtlalpha1.0/decode_train_dev_decode_ctcweight1.0_l
 exp/train_nodev_pytorch_train_mtlalpha1.0/decode_test_decode_ctcweight1.0_lm_word7184/result.txt:|  SPKR         |  # Snt    # Wrd   |  Corr       Sub      Del       Ins       Err    S.Err   |
```

Comment:
Can you follow `RESULTS` with the other one (e.g., https://github.com/espnet/espnet/blob/master/egs/librispeech/asr1/RESULTS.md)?
- [ ] change it from text to markdown `RESULTS.md`
- [ ] change the order of the results starting from new one
- [ ] add more system information to run the recipe (see https://github.com/espnet/espnet/blob/master/CONTRIBUTING.md#how-to-contribute-to-espnet)
- [ ] upload your models and put a model link to `RESULTS.md`

Reply:
I'll send our Google drive link invitation later.

Reply:
Sure! My policy when working on an existing recipe is to follow the same style, that's why I didn't touch the results file. I'll update the file and add model to google drive!

Reply:
Oh, I see.
Thanks.

Reply:
Which model(s) do you want me to upload? Also, should I retrain default config to add missing infos and WER report?

Reply:
Oh and I'll try to to work a bit on the config files later when I have time for that. I didn't really tune each network so comparison is a bit pointless right now